lib/CleantalkSP/SpbctWP/Scanner/ScannerQueue.php

```diff
@@ -1134,6 +1139,11 @@ public function heuristic_analysis($status = 'UNKNOWN,MODIFIED,OK,INFECTED', $of
                         ) : 'NULL')
                         . ' WHERE fast_hash = \'' . $file['fast_hash'] . '\';'
                     );
+
```

Comment:
Please, add the todo to make one bulk SQL-request instead of tiny multiple requests.